web-registry/src/lib/ecocredit/api.ts

```diff
@@ -217,6 +206,70 @@ export const getEcocreditTxs = async (): Promise<TxResponse[]> => {
   });
 };
 
+export const getBridgedEcocreditsForAccount = async (
+  addr?: string,
+  sanityCreditClassData?: AllCreditClassQuery,
+): Promise<BridgedEcocredits[] | undefined> => {
+  if (addr) {
+    // Since bridged ecocredits are canceled on Regen Ledger,
+    // the only way to get them is to fetch txs by event and loop through the tx messages.
+    // Txs can have multiple messages and MsgBridge messages can have multiple credits
+    // so we can't really support pagination.
+    const res = await getTxsByEvent({
+      events: [
+        `${messageActionEquals}'/${MsgBridge.$type}'`,
+        `message.sender='${addr}'`,
+      ],
+      orderBy: OrderBy.ORDER_BY_DESC,
+    });
+
+    const bridgedCredits: BridgedEcocredits[] = [];
+    const batchesMap = new Map<string, IBatchInfoWithClassProject>();
```

Comment:
using a map so we don't fetch credit class and project info associated to a same batch multiple times, we should try to do something similar when using `getClassProjectForBatch` but I guess I'll leave it for another performance related task